.github/workflows/R-CMD-check.yaml

```
@@ -89,7 +93,7 @@ jobs:
     name: 'gpu'
 
     container:
-      image: nvidia/cuda:10.2-cudnn7-devel-ubuntu18.04
+      image: nvidia/cuda:10.2-cudnn8-devel-ubuntu18.04
```

I think it makes sense to also update the CUDA toolkit version since we are here.

```suggestion
      image: nvidia/cuda:11.1.1-cudnn8-devel-ubuntu18.04
```